controllers/controller.js

```diff
@@ -61,41 +61,87 @@ exports.searchLocationPage = function (req, res, next) {
 }
 
 exports.postCreateStoryPage = function (req, res, next) {
-  newStoryData = req.body;
-  res.redirect('/create-story');
+  console.log('client post works');
+  // newStoryData = req.body;
+
+  // Create a stories array in session:
+  if (!req.session.stories) {
+    req.session.stories = [];
+  }
+
+  // Create id for new story:
+  var id = shortid.generate();
+
+  // Create a story object, which we will fill in later:
+  var story = {
+    "id": id,
+    "key": null,
+    "title": null,
```

Comment:
We hebben nu geen title meer toch op de create-story page? Of ga je dat in een andere branch oplossen?